op-challenger/game/fault/agent.go

```diff
@@ -129,6 +143,15 @@ func (a *Agent) tryResolveClaims(ctx context.Context) error {
 
 	var resolvableClaims []int64
 	for _, claim := range claims {
+		if a.selective {
+			a.log.Trace("Selective claim resolution, checking if claim is incentivized", "claimIdx", claim.ContractIndex)
+			isUncounteredClaim := slices.Contains(a.claimants, claim.Claimant) && claim.CounteredBy == common.Address{}
+			ourCounter := slices.Contains(a.claimants, claim.CounteredBy)
```

Comment:
Why attempt to resolve a claim that was already resolved by someone else? I guess this is how it works without `selective`. Though we should be filtering on claims that are uncountered regardless of `selective`.

Reply:
If there is no countered by, the bond recipient is the claimant, and if it is our claim, we would then receive that bond. Thus we are incentivized to resolve the claim for which it is our own bond.

Reply:
Quite so.
OTOH, if the selected claim's `counteredBy` is in `claimants`, then we don't skip it per L150. I was originally thinking we'd waste gas in the resolveClaim attempt, but we first try calling it which will fail so this is fine. (We probably should filter those claims anyways to avoid log errors when they fail)